src/scaler/daemonscaler/mod.rs

```diff
@@ -0,0 +1,1038 @@
+            None
+        };
+
+        let actor_id = actor.as_ref().map(|actor| actor.id.as_str());
+
+        let mut spread_status = vec![];
+
+        trace!(spread = ?self.config.spread_config.spread, ?actor_id, "Computing commands");
+        let commands = self
+            .config
+            .spread_config
+            .spread
+            .iter()
+            .filter_map(|spread| {
+                let eligible_hosts = eligible_hosts(&hosts, spread);
+                if !eligible_hosts.is_empty() {
+                    // Create a list of (host_id, current_count) tuples
+                    // current_count is the number of actor instances that are running for this spread on this host
+                    let actors_per_host = eligible_hosts
+                        .into_keys()
+                        .map(|id| {
+                            let count = actor
+                                .as_ref()
+                                .and_then(|actor| {
+                                    actor.instances.get(&id.to_string()).map(|instances| {
+                                        instances
+                                            .iter()
+                                            .filter(|instance| {
+                                                spreadscaler_annotations(&spread.name, self.id())
+                                                    .iter()
+                                                    .all(|(key, value)| {
+                                                        instance
+                                                            .annotations
+                                                            .get(key)
+                                                            .map(|v| v == value)
+                                                            .unwrap_or(false)
+                                                    })
+                                            })
+                                            .count()
+                                    })
+                                })
+                                .unwrap_or(0);
+                            (id, count)
+                        })
+                        .collect::<Vec<(&String, usize)>>();
+
+                    Some(
+                        actors_per_host
+                            .iter()
+                            .filter_map(|(host_id, current_count)| {
+                                // Here we'll generate commands for the proper host depending on where they are running
+                                match current_count.cmp(&self.config.spread_config.replicas) {
+                                    Ordering::Equal => None,
+                                    // Start actors to reach desired replicas
+                                    Ordering::Less => {
+                                        // Right now just start on the first available host. We can be smarter about it later
```

Comment:
Is this comment a copy/paste leftover? We start on all matching hosts so I figure this doesn't apply

Reply:
Yes, good catch